### PR TITLE
Updated 'Cookies and analytics' section

### PR DESCRIPTION
**Pull request by thomasjhughes1**
I've added some new information about sharing data for fraud and threat intelligence reasons to the 'Cookies and analytics' section.

I've also rewritten the rest of the GOV.UK Verify privacy notice to make sure it meets GDS style.

### CODE REVIEW

File: app/views/static/privacy_notice.en.html.erb [hunk 1]
```diff
@@ -9,160 +9,223 @@ end %>
...
-      Cabinet Office
-      <br>
-      70 Whitehall
-      <br>
-      London SW1A 2AS
-    </address>
-    </p>
-    <p>The DPO provides independent advice and monitoring of our use of personal information.</p>
+    <p>Last updated: 20 September 2019</p>
+ <h2 class="heading-large" id="who-we-are">Who we are</h2>
```
Comment (jakubmiarka) [inline app/views/static/privacy_notice.en.html.erb]: The indentation is incorrect

Comment (idrop) [reply app/views/static/privacy_notice.en.html.erb]: I've just pushed a change to the test

Comment (thomasjhughes1) [reply app/views/static/privacy_notice.en.html.erb]: I have fixed the indentation. 

I don't have the Welsh translation yet. We're planning on doing another small update to the privacy notice next week that we're still waiting for clarification on. I'll wait until we have everything ready before I contact the Welsh translation unit to save sending them 2 lots of work.

Comment (jakubmiarka) [reply app/views/static/privacy_notice.en.html.erb]: Thanks, now even the diff is much nicer and easier to compare.

OK, just be aware that the policies will have a different publish dates and content, which might not be ideal.

File: app/views/static/privacy_notice.en.html.erb [hunk 2]
```diff
@@ -9,66 +9,86 @@ end %>
...
+    <ol class="list-bullet list">
+      <li>sign in to a service with GOV.UK Verify</li>
+      <li>sign in to a service with a digital identity from an EU country</li>
+      <li>send us questions or feedback</li>
+    </ol>
 
-    <h3 class="heading-medium" id="how-personal-data-is-used-when-signing-in-with-gov.uk-verify">How personal data is used when signing in with GOV.UK Verify</h3>
-    <p>GOV.UK Verify provides a ‘hub’ between services and companies which verify users’ identities on behalf of the government. These are known as ‘certified companies’. They have met government and industry standards to provide identity assurance services as part of GOV.UK Verify.</p>
-    <p>After you’ve been verified by a company, they will send your personal data to the hub. The hub will then send it to the government service that you want to use.</p>
+    <h3 class="heading-large" id="when-you-sign-in-with-govuk-verify">When you sign in with GOV.UK Verify</h3>
```
Comment (jakubmiarka) [inline app/views/static/privacy_notice.en.html.erb]: Just noticed, `<h3>` should have `heading-medium` class, to be consistent and syntactically correct it should be aligned with the rest of the document (h1 - xlarge, h2 - large, h3 - medium etc...)

**Pull request merged**